htdocs/user/class/userbankaccount.class.php

```diff
@@ -165,17 +168,23 @@ public function fetch($id, $ref = '', $userid = 0)
 			return -1;
 		}
 
-		$sql = "SELECT rowid, fk_user, entity, bank, number, code_banque, code_guichet, cle_rib, bic, iban_prefix as iban, domiciliation, proprio,";
-		$sql .= " owner_address, label, datec, tms as datem";
-		$sql .= " FROM ".$this->db->prefix()."user_rib";
+		$sql = "SELECT ur.rowid, ur.fk_user, ur.entity, ur.bank, ur.number, ur.code_banque, ur.code_guichet, ur.cle_rib, ur.bic, ur.iban_prefix as iban, ur.domiciliation, ur.proprio";
+		$sql .= ", ur.owner_address, ur.label, ur.datec, ur.tms as datem";
+		$sql .= ', ur.currency_code, ur.state_id, ur.fk_country as country_id';
+		$sql .= ', c.code as country_code, c.label as country';
+		$sql .= ', d.code_departement as state_code, d.nom as state';
+		$sql .= " FROM ".$this->db->prefix()."user_rib as ur";
+		$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'c_country as c ON ur.fk_country=c.rowid';
```

Comment:
Use  $this->db->prefix()  instead of  MAIN_DB_PREFIX  here (method seems used by tools that uses connection on several Dolibarr).